examples/tf/trpo_cubecrash.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+"""
+This is an example to train a task with TRPO algorithm.
+
+Here it runs CubeCrash-v0 environment with 100 iterations.
+"""
+import gym
+
+from garage.envs import normalize
+from garage.experiment import run_experiment
+from garage.tf.algos import TRPO
+from garage.tf.baselines import GaussianCNNBaseline
+from garage.tf.envs import TfEnv
+from garage.tf.experiment import LocalTFRunner
+from garage.tf.policies import CategoricalConvPolicy
+
+
+def run_task(snapshot_config, *_):
+    """Run task."""
+    with LocalTFRunner(snapshot_config=snapshot_config) as runner:
+        env = TfEnv(normalize(gym.make('CubeCrash-v0')))
+        policy = CategoricalConvPolicy(env_spec=env.spec,
```

Comment:
Might be better to use `CNN` for convention, so it would be `CategoricalCNNPolicy`. Please also edit other occurrence.